src/shogun/evaluation/ROCEvaluation.cpp

```diff
@@ -19,15 +19,16 @@ CROCEvaluation::~CROCEvaluation()
 
 float64_t CROCEvaluation::evaluate(CLabels* predicted, CLabels* ground_truth)
 {
-	return evaluate_roc(predicted,ground_truth);
+	ASSERT(predicted->get_label_type()==LT_BINARY)
```

Comment:
Can we use REQUIRE here and give the user information of what is wrong?
https://github.com/shogun-toolbox/shogun/wiki/Assertions